### PR TITLE
When Smarty returns error 422, parse and include the message in the SDK response.

### PR DESCRIPTION
**Pull request by waleo**
If SmartyAPI returns error messages for status 422, merge and ruturn those errors to the consumer of the SDK. This can happen if a bad country is specified for international api verification. This change allows the consumer of the SDK to receive a meaningful message about the error field so users can take the appropriate action.

Supercat source: SERV-1379

### CODE REVIEW

File: .gitignore
Comment (brucew) [inline .gitignore:1]: This should probably be a separate commit

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,5 @@
 .idea
 coverage
 *.gem
+.byebug_history
+.ruby-version
```
Comment (brucew) [inline .gitignore]: Are you sure `.ruby-version` should be included here?

**Review by brucew — CHANGES_REQUESTED** [review]
I just have a question about the `.gitignore` changes but otherwise good work in a difficult situation

File: .gitignore
Comment (waleo) [reply .gitignore:1]: I removed the changes to .gitignore

**Review by brucew — APPROVED** [review]
Again, nice work under restricted circumstances!
